force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,39 @@
+trigger OpportunityTrigger on Opportunity (before update, before delete) {
+    if (Trigger.isBefore){
+        if(Trigger.isUpdate){
+            Set<Id> accountIds = new Set<Id>();
+            for(Opportunity opp : Trigger.new){
+                if(opp.Amount < 5000) {
+                    opp.addError('Opportunity amount must be greater than 5000');
```

Comment:
Put the string error message in a top level variable so it can be changed easily.